types/react-native-safe-area-context/CompatNativeSafeAreaProvider.js.flow

```diff
@@ -0,0 +1,6 @@
+/* @flow
+ * @generated by TsFlower
```

Comment:
Yeah, Flowgen does that.

When experimenting with changes to Flowgen, I found that it caused a lot of churn -- you try a new version that should only make a few changes, and it touches every one of the many different files because it updates the version number. At one point I was hacking my local copy of Flowgen to lie about its version number to avoid that churn.

So, better to leave that information to the yarn.lock.